src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/SetCustomerFormSubscriber.php

```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-namespace Sylius\Bundle\UserBundle\Form\EventSubscriber;
+namespace Sylius\Bundle\CoreBundle\Form\EventSubscriber;
```

Comment:
I missed `$customer->getUser()`! No need for default implementation, let's leave it in `Core` ;)